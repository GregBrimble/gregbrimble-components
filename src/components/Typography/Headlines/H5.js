import React from 'react';

import styled from 'styled-components';
import {HeadlineSmall} from './shared';

const H5 = props => <HeadlineSmall as={styled.h5`
    font-size: 1.5rem;
    line-height: 2rem;
`} {...props} />;

export default H5;
