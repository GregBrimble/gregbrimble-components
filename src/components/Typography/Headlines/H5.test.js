import React from "react"
import H5 from "./H5"
import renderer from "react-test-renderer"

describe("H5", () => {
  it("renders properly", () => {
    const tree = renderer.create(<H5>Headline 1</H5>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
