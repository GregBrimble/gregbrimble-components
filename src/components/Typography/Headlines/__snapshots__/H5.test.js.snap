// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H5 renders properly 1`] = `
<h5
  className="sc-bdVaJa sc-bwzfXH cSXMnb sc-htpNat iBnHJT"
>
  Headline 1
</h5>
`;
